### DISCUSSION TITLE
Começando do Zero

### DISCUSSION THREAD

**Discussion by gustavocarl** [frontendbr/forum#2402] [Discussão]
Olá pessoal, tudo bem?
Estou começando agora a programar, porém estou com dúvidas.
Estou estudando HTML, CSS e JavaScript, se alguém tiver alguma dica que possa me ajudar serei grato.

**Comment by mstuttgart** [comment]
@gustavocarl olá. Tudo tranquilo? 

Eu já trabalho com programação tem alguns anos (sou backend Python), mas recentemente comecei a estudar Javascript também. O que eu posso te falar é: estude com calma, sem pressa. Eu tenho feito os exercicios desse site: https://exercism.org/
Está me ajudando demais :) 

No mais, boa sorte com os estudos! 😁

**Reply by gustavocarl** [reply]
Obrigado pela dica

**Comment by meow-os** [comment]
Opa, eae @gustavocarl então mano, eu também estou começando agora e estou postando tudo que faço de exercicio e coisas no meu repositório do meu perfil. Atualmente estou fazendo o curso web moderno da udemy, se quiser o link posso deixar, ele é um curso muito bom, se você criar conta na udemy consegue um cupom que paga uns 40 reais pelo curso. Vale a pena!

**Reply by gustavocarl** [reply]
Se puder deixar o link do curso agradeço, será de muito boa ajuda.
Obrigado pela dica Marco
